### PR TITLE
Word Cloud: style refactoring & save selection

### PR DESCRIPTION
**Pull request by kernc**
##### Issue
Fixes https://github.com/biolab/orange3-text/issues/215.

##### Description of changes
[wordcloud2](https://github.com/timdream/wordcloud2.js) is updated. Selection is saved in workflow. Colors  and word tilts are adapted.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: orangecontrib/text/widgets/owwordcloud.py [hunk 1]
```diff
@@ -90,15 +37,18 @@ class OWWordCloud(widget.OWWidget):
 
     graph_name = 'webview'
 
-    selected_words = settings.ContextSetting(SelectedWords('whatevar (?)'))
+    selected_words = settings.Setting(set(), schema_only=True)
```
Comment (nikicc) [inline orangecontrib/text/widgets/owwordcloud.py]: Selection is not restored for me, when I open the schema.

**Comment by nikicc** [comment]
@ajdapretnar when you're back, please check the how do these fonts appear on a windows machine 😄

**Comment by ajdapretnar** [comment]
Am I the only one who doesn't like the new visualization? It is very unclear to me. Fonts are so thin and small, colors so uniform... Also the default tilt is quite annoying.

I seriously could scan word frequencies much easier with the old visualization, but if everyone is on board with this, then fine. I'll check how it looks on Windows first thing tomorrow.

**Comment by ajdapretnar** [comment]
Also when I set the tilt to 0 and open a new Word Cloud, I am not getting this as a setting (at 45 degrees once again). 😞

**Comment by kernc** [comment]
I could live with font appearing bolder,  but it was @nikicc's choice. I styled the word cloud (colors and default tilt) on [Vega's wordcloud](https://vega.github.io/vega-editor/?mode=vega&spec=wordcloud), which @BlazZupan said trumps ours many times over.

Can't reproduce you observed settings issue.

**Comment by ajdapretnar** [comment]
> @ajdapretnar when you're back, please check the how do these fonts appear on a windows machine 

Just a final comment - it works ok on Win as well. :) Now I'll stfu.

**Pull request merged**